src/config/validation.spec.ts

```
@@ -25,10 +25,14 @@ describe('validationSchema', () => {
       STORAGE_BUCKET: 'storage_bucket',
       MESSAGING_SENDER_ID: 'messaging_sender_id',
       APP_ID: 'app_id',
+      RECAPTCHA_SECRET_KEY: 'recaptcha_secret_key',
     };
 
+    const result = validationSchema.validate(validConfig, { abortEarly: false });
+
     const result = firebaseValidationSchema.validate(validConfig);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'result'. Consider to delete it or rename it.

'result' is defined here:


(lint/suspicious/noRedeclare)